Vagrantfile

```diff
@@ -54,11 +55,6 @@ else
     curl -sSL https://storage.googleapis.com/golang/go#{go_version}.linux-amd64.tar.gz  | sudo tar -xz -C /usr/local
 fi
 
-if [[ $# -gt 9 ]] && [[ $9 != "" ]]; then
```

Comment:
This was probably not having the intended outcome.

This was putting `export classic_mode` before this change in the `/etc/profile.d/envvar.sh` file. This isn't helpful at all.

I've removed all this code. This doesn't affect the resulting `envvar.sh`.